pe.c

```diff
@@ -1079,6 +1081,16 @@ handle_image (void *data, unsigned int datasize,
 			return EFI_UNSUPPORTED;
 		}
 
+		efi_status = parse_sbat_var(&sbat_var_entries);
+		/* we get EFI_SUCCESS only if variable is fully parsed,
+		   and EFI_NOT FOUND means do not perform verificaiton
+		   continue with EFI_SUCCESS
```

Comment:
There's two typos here: verification and EFI_NOT_FOUND.

But I would maybe change this comment to something like:

```
/*
 * Until a SBAT variable is installed into the systems, it is expected that
 * attempting to parse the variable will fail with a EFI_NOT_FOUND error.
 *
 * Do not consider that error fatal for now.
 */
```